std/experimental/allocator/building_blocks/affix_allocator.d

```diff
@@ -88,7 +88,7 @@ struct AffixAllocator(Allocator, Prefix, Suffix = void)
     private template Impl()
     {
 
-        size_t goodAllocSize(size_t s)
+        @safe size_t goodAllocSize(size_t s)
```

Comment:
Imho I think that this function should be safe and thus I thought we should enforce it upon the parent.